src/test/java/HandleGetRequestTest.java

```diff
@@ -2,9 +2,9 @@
 import java.util.*;
 import java.io.IOException;
 
-public class HttpHandlerSelectorTest extends junit.framework.TestCase {
+public class HandleGetRequestTest extends junit.framework.TestCase {
 
-  HttpHandlerSelector handler;
+  HandleGetRequest getRequest;
```

Comment:
You mean the name of the instance of the class is confusing, and should probably be named the same as the class, handleGetRequest??